src/Apps/Search/Routes/Articles/SearchResultsArticles.tsx

```diff
@@ -71,7 +72,9 @@ export class SearchResultsArticlesRoute extends React.Component<
         {articles.map((article, index) => {
           return (
             <Box key={index}>
-              {article.displayLabel}
+              <GenericSearchResultItem item={article} index={index} />
+              <Spacer mb={3} />
+              <Separator />
```

Comment:
#minor nit, we probably don't need `<Separator />` for the _last_ element to be rendered.